src/main/java/seedu/address/model/Entity/Mentor.java

```diff
@@ -97,24 +85,19 @@ public void setSubject(SubjectName subject) {
     @Override
     public HashMap<String, String> viewDetailed() {
         HashMap<String, String> result = new HashMap<>();
-        Stream<String> teamStream = teams.stream().map(team -> team.toString());
-        String teamsString = Arrays.toString(teamStream.toArray());
 
         result.put("name", getName().toString());
```

Comment:
Also, should we have an enum or constants to hold field names? i.e. `Enum.NAME` instead of `name`.

Reply:
Oh actually this has been deleted cos Mentor no longer has teams

Reply:
Oh I meant when you do result.put("name", ...), instead of using `"name"`, we should have an enum or constant for `"name"` and the other fields.

Reply:
Hmm can is can. One enum for all entities or 1 for each? Though i think we should keep this for now because the code is not client facing and that the keys are always equals to the entity attributes. Wdyt?

Reply:
For now should be fine. But later I think we should change cuz they said don't use magic strings. I was thinking of having one class with all the fields as constants. Inside the Entity package, we can make one Field class (or smth) that keeps all.

Reply:
Cool ok will create one enum then